src/store/index.js

```diff
@@ -5,11 +5,19 @@ Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
-    savedParks: []
+    savedParks: [],
+    searchResults: [],
+    geolocation: null
   },
```

Comment:
I know our app isn't huge, so maybe geolocation doesn't NEED to be in the store, but I think it is probably good practice though.

Reply:
I agree.  I think since we haven't found a way to pass it as props from App (given App only explicitly renders the header), this is our best solution for now.  I figure if geolocation is only needed in one component, we could do a simple `provide / inject` and store it in App as data.

Reply:
Agreed. We will potentially want to have access to it in ResultsMap in order to center the map on our current location. Having it in the store definitely lends itself better to scalability!